pkg/deployments/tasks/20230206-erc4626-linear-pool-v3/test/task.fork.ts

```diff
@@ -0,0 +1,352 @@
+import hre from 'hardhat';
+import { expect } from 'chai';
+import { Contract } from 'ethers';
+import { setCode } from '@nomicfoundation/hardhat-network-helpers';
+import * as expectEvent from '@balancer-labs/v2-helpers/src/test/expectEvent';
+import { bn, fp, FP_ONE } from '@balancer-labs/v2-helpers/src/numbers';
+import { MAX_UINT256 } from '@balancer-labs/v2-helpers/src/constants';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/dist/src/signer-with-address';
+
+import { impersonate, getForkedNetwork, Task, TaskMode, getSigners } from '../../../src';
+import { describeForkTest } from '../../../src/forkTests';
+import { deployedAt, getArtifact } from '@balancer-labs/v2-helpers/src/contract';
+
+export enum SwapKind {
+  GivenIn = 0,
+  GivenOut,
+}
+
+//TODO remove skip when we have build-info
+describeForkTest.skip('ERC4626LinearPoolFactory', 'mainnet', 16550500, function () {
+  let owner: SignerWithAddress, holder: SignerWithAddress, other: SignerWithAddress;
+  let factory: Contract, vault: Contract, mainToken: Contract;
+  let rebalancer: Contract;
+
+  let task: Task;
+
+  const frxEth = '0x5E8422345238F34275888049021821E8E08CAa1f';
```

Comment:
Do we want to test USDT here to check the fixed rebalancer?

Reply:
Integrations is doing this. First they need to add it to their version of the Rebalancer. I suggested using our USDTMock so it can be more easily (unit) tested.

Reply:
Sounds good, let's close this then.